src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -244,6 +254,11 @@ def process_report(processes, most_recent_node, levelname, indent_size, max_dept
     from aiida.cmdline.utils.common import get_calcjob_report, get_process_function_report, get_workchain_report
     from aiida.orm import CalcFunctionNode, CalcJobNode, WorkChainNode, WorkFunctionNode
 
+    if not (processes) and not (most_recent_node):
```

Comment:
```suggestion
    if not processes and not most_recent_node:
```

Reply:
Please apply also in all other commands. These parentheses are redundant

Reply:
done